lib/unified_health_data/service.rb

```diff
@@ -0,0 +1,148 @@
+# frozen_string_literal: true
+
+require 'common/client/base'
+require_relative 'configuration'
+require_relative 'models/lab_or_test'
+
+module UnifiedHealthData
+  class Service < Common::Client::Base
+    configuration UnifiedHealthData::Configuration
+
+    def initialize(user)
+      super()
+      @user = user
+    end
+
+    def get_medical_records(start_date:, end_date:)
```

Comment:
Did we want to change this to something like `get_labs_and_tests`?

Reply:
yup good call, done!